docs/kratos/organizations/organizations.mdx

```diff
@@ -3,6 +3,8 @@ id: organizations
 title: B2B Single Sign-On
```

Comment:
In the sidebar SAML is no longer visible now - can we change that?

Reply:
changed it to `B2B SSO & SAML`